android/build.gradle

```diff
@@ -44,23 +44,26 @@ rootProject.allprojects {
 apply plugin: 'com.android.library'
 
 def DEFAULT_PLAY_SERVICES_BASE_VERSION = '16.0.1'
-def READER_SDK_VERSION = '[1.5.1, 2.0)'
+def READER_SDK_VERSION = '[1.6.1, 2.0)'
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion 32
 
     defaultConfig {
-        minSdkVersion 21
-        targetSdkVersion 29
+        minSdkVersion 24
+        targetSdkVersion 32
```

Comment:
This need to be downgraded to 31 as well.

Reply:
fixed